internal/app/api/v1/graphql.go

```diff
@@ -0,0 +1,28 @@
+package v1
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/99designs/gqlgen/graphql/playground"
+
+	"github.com/kubeshop/testkube/internal/config"
+	"github.com/kubeshop/testkube/internal/graphql"
+	"github.com/kubeshop/testkube/pkg/log"
+)
+
+// RunGraphQLServer runs GraphQL server on go net/http server
+// There is an issue with gqlgen and fasthttp server
+func (a *TestkubeAPI) RunGraphQLServer(
+	ctx context.Context,
+	cfg *config.Config,
+) error {
+	srv := graphql.GetServer(a.Events.Bus, a.ExecutorsClient)
+
+	http.Handle("/", playground.Handler("GraphQL playground", "/query"))
+	http.Handle("/query", srv)
+
+	log.DefaultLogger.Infow("running GraphQL server", "port", cfg.GraphqlPort)
+
+	return http.ListenAndServe(":"+cfg.GraphqlPort, nil)
```

Comment:
If we want to use it on the front end, the GraphQL server should be exposed under the same port - either directly or proxied.

Otherwise, we would need to pass two URLs to the front-end - `apiEndpoint` and `gqlEndpoint`.

Reply:
@rangoo94 it's not possible on this level, as current GraphQL implementation in go doesn't handle FastHTTP well .

Reply:
So we need to split it, but I think it should be possible to merge it on ingress level (ar least I hope so ;)